tock/tock.go

```diff
@@ -37,7 +39,7 @@ type APIPages struct {
 //the Reporting Period Audit list endpoint
 type ReportingPeriodAuditList struct {
 	APIPages
-	ReportingPeriods []ReportingPeriod `json:"results"`
+	ReportingPeriods []ReportingPeriod
```

Comment:
Here’s another one @toolness this used to return a JSON object with a results property. Though now it just returns an Array.

Reply:
Looks like line 49 also needs this update